casepro/backend/junebug.py

```diff
@@ -463,3 +467,44 @@ def received_junebug_message(request):
         created_on=datetime.now(), has_labels=True)
 
     return JsonResponse(msg.as_json())
+
+
+@csrf_exempt
+def receive_identity_store_optout(request):
+    '''Handles optout notifications from the Identity Store.'''
+    if request.method != 'POST':
+        return JsonResponse({'reason': 'Method not allowed.'}, status=405)
+
+    try:
+        data = json.loads(request.body)
+    except ValueError as e:
+        return JsonResponse(
+            {'reason': 'JSON decode error', 'details': e.message}, status=400)
+
+    identity_id = data.get('identity_id')
+    optout_type = data.get('optout_type')
+
+    syncer = IdentityStoreContactSyncer()
+
+    # The identity store currently doesn't specify the response format or do
+    # anything with the response.
+
+    # I don't know how to get the org
+    local_contact = syncer.fetch_local(org, identity_id)
```

Comment:
Good question.

I've realized that we currently load all identities for every organization (i.e. we ignore the `org` parameter in `pull_contacts` on the Junebug backend). This means that for now we really only cater for a single organization. I suggest we thus use `org = Org.objects.first()` for the moment and create an issue for properly supporting `orgs` later (this will either require a separate identity store per Org, or a detail field in the identity saying which Org the identity belows to).